OpenRA.Mods.Cnc/CncLoadScreen.cs

```diff
@@ -41,7 +43,9 @@ public override void Init(Manifest m, Dictionary<string, string> info)
 			r = Game.Renderer;
 			if (r == null) return;
 
-			sheet = new Sheet(Platform.ResolvePath(loadInfo["Image"]));
+			var image = Platform.ResolvePath(info["Image"]);
+			GlobalFileSystem.Mount(Path.GetDirectoryName(image));
```

Comment:
This looks like a workaround / bandaid for something bogus.  Why are all these additional mounts needed?